test/LocalizationWebsite/LocalizationWebsite.csproj

```diff
@@ -21,4 +21,10 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.2.0-*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Update="Resources\StartupCustomCulturePreserved.en-US.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
VS seems to be automatically adding these when you create a new resource file even though things work fine without them. I've given feedback on that through the in-VS mechanism and will keep a closer eye on it in the future.